cmd/main.go

```diff
@@ -1,59 +1,60 @@
 package main
 
 import (
-	_ "embed"
 	"fmt"
+	"log"
 	"os"
-	"regexp"
 
-	"github.com/docopt/docopt-go"
+	"github.com/urfave/cli/v2"
 )
 
-//go:embed Readme.md
-var readme string
-
-func helpText() string {
-	usageRegexp := regexp.MustCompile("(?s)<!-- begin usage -->\n\n```\n(.*)\n```\n\n<!-- end usage -->")
-	usageText := usageRegexp.FindStringSubmatch(readme)[1]
-	return fmt.Sprintf(helpTextTemplate, usageText)
+func main() {
+	app := scipApp()
+	err := app.Run(os.Args)
+	if err != nil {
+		log.Fatal(err)
+	}
 }
 
-const helpTextTemplate string = `Semantic Code Intelligence Protocol CLI.
-
-%s
-
-For more details, see the project README: https://github.com/sourcegraph/scip
-`
+func commands() []*cli.Command {
+	convert := convertCommand()
+	snapshot := snapshotCommand()
+	stats := statsCommand()
+	return []*cli.Command{&convert, &snapshot, &stats}
+}
 
-func bailIfError(err error) {
-	if err != nil {
-		os.Stderr.WriteString(err.Error())
-		os.Exit(1)
+func scipApp() cli.App {
+	var versionFlag bool
+	app := cli.App{
+		Name:        "scip",
+		Usage:       "SCIP Code Intelligence Protocol CLI",
+		Description: "For more details, see the project README at:\n\n\thttps://github.com/sourcegraph/scip",
+		Flags: []cli.Flag{
+			&cli.BoolFlag{
+				Name:        "version",
+				Usage:       "Print the current version and exit.",
+				Destination: &versionFlag,
+				Aliases:     []string{"v"},
+			},
+		},
+		Action: func(c *cli.Context) error {
+			if versionFlag {
+				fmt.Println("0.1.0")
+				os.Exit(0)
+			}
+			// FIXME: What is the right way to print help text and error here?
```

Comment:
I should file an issue to fix this if we don't know a good way to fix this right now.

Reply:
https://github.com/sourcegraph/scip/issues/49